frontend/src/_components/Header.jsx

```diff
@@ -30,6 +30,10 @@ export const Header = function Header({ switchDarkMode, darkMode }) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [avatar_id]);
 
+  const version = config.currentVersion;
+  const [currentVersion, setCurrentVersion] = useState(version);
```

Comment:
Hey @manishkushare , line 33 to 35 can be reduced to a single statement:
```
const currentVersion = useMemo(() => config.currentVersion, [config.currentVersion]);
```

Reply:
That is, you can combine the useState and useEffect into a single useMemo. Can also eliminate the unnecessary assignment at line 33 this way.